internal/api/front/perses/src/app/project/dashboard/dashboardfeed.service.ts

```diff
@@ -0,0 +1,38 @@
+// Copyright 2021 Amadeus s.a.s
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import { Injectable } from '@angular/core';
+import { HttpClient } from '@angular/common/http';
+import { Observable } from 'rxjs';
+import { DashboardSpec } from './dashboard.model';
+import { DashboardFeedModel } from './dashboardfeed.model';
+import { UrlBuilderUtil } from '../../shared/utils/url-builder.util';
+import { ErrorHandlingService } from '../../shared/service/error-handling.service';
+import { catchError } from 'rxjs/operators';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class DashboardFeedService {
+  private datapointsResource = 'feed/sections';
```

Comment:
Datapoints, sections, dashboard feed, maybe it deserves some comments I confess that I don't know anything but I find the naming not self explanatory.

Reply:
Actually it maps to backend concepts named the same, which are documented

Still I can indeed add some comments

Reply:
I would do more something like that : 

```typescript
  private readonly resource = 'feed';

  constructor(private readonly http: HttpClient, private readonly errorHandler: ErrorHandlingService) {
  }

  feedSections(spec: DashboardSpec): Observable<DashboardFeedModel[]> {
    const url = new UrlBuilderUtil().setResource(`${this.resource}/sections`);
    return this.http.post<DashboardFeedModel[]>(url.build(), request).pipe(catchError(this.errorHandler.handleHTTPError));
  }
```

because in the same service you will have the method to get the value of the different variable too

Reply:
Indeed I find it more self explanatory and pasting the API what @Nexucis proposes. I was not understandning why "section" wording was translated to "datapoints"

Reply:
@celian-garcia Oh ok I missunderstood your first comment actually, yes looks like I forgot to do a second pass on this part to better name things

@Nexucis actually the method was named "feedDashboard" before, until I renamed it to a generic "get" to be better aligned with the other services, but yeah didnt think about the coming variables so I will change it back ^^